webapp/channels/src/components/emoji_picker/emoji_picker.tsx

```diff
@@ -391,8 +391,8 @@ const EmojiPicker = ({
             />
             {areSearchResultsEmpty ? (
                 <NoResultsIndicator
-                    variant={NoResultsVariant.ChannelSearch}
-                    titleValues={{channelName: `"${filter}"`}}
+                    variant={NoResultsVariant.Search}
+                    titleValues={{channelName: `“${filter}”`}}
```

Comment:
Instead of passing the quotation marks into the translation string as part of the value, we should actually be making them part of the translation string since some languages use different characters for them like French which uses «these ones».

I can split that off into a separate ticket if you'd like though since it's not directly related to this one

Reply:
@matthewbirtch Could you please share your opinion as well?

Reply:
what @hmhealey is suggesting makes sense to me

Reply:
Ticket to do that: https://mattermost.atlassian.net/browse/MM-55416